README.md

```diff
@@ -933,6 +933,7 @@ _Games, game servers and control panels._
 - [Chamilo LMS](https://chamilo.org/) - Chamilo LMS allows you to create a virtual campus for the provision of online or semi-online training. ([Source Code](https://github.com/chamilo/chamilo-lms)) `GPL-3.0` `PHP`
 - [edX](https://www.edx.org/) - The Open edX platform is open-source code that powers edX.org. ([Source Code](https://github.com/edx/)) `AGPL-3.0` `Python`
 - [ILIAS](http://www.ilias.de) - ILIAS is the Learning Management System that can cope with anything you throw at it. ([Demo](http://demo.ilias.de), [Source Code](https://github.com/ILIAS-eLearning/ILIAS)) `GPL-3.0` `PHP`
+- [LibreLingo](https://librelingo.app/) - LibreLingo is a community-owned language-learning platform ([Demo](https://librelingo.app/course/spanish-from-english/), [Source Code](https://github.com/kantord/LibreLingo)) `LGPL-3.0` `JavaScript/Python`
```

Comment:
```diff
-`JavaScript/Python`
+`Python`
```

Reply:
It's really not just Python though :thinking: If I really have to put one, I guess I would put JavaScript then :thinking:

Reply:
> https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/.github/PULL_REQUEST_TEMPLATE.md  
> The Language tag is the main **server-side** requirement for the software

Reply:
in this case... LibreLingo can be hosted as a static site :thinking: Strictly speaking there isn't a main server-side requirement.

If you want to build the static site for yourself it would be nodejs, but the pre-built static site is available as well :thinking: ]

I guess let's put `JavaScript`? :thinking: